plugins/com.google.cloud.tools.eclipse.appengine.ui/about.properties

```diff
@@ -0,0 +1,8 @@
+blurb=Google Cloud Tools for Eclipse\n\
+Version: {featureVersion}\n\
+\n\
+Copyright 2016-2017 Google Inc.\n\
+\n\
+Cloud Tools for Eclipse is a Google-sponsored plugin that adds \
+support for the App Engine standard environment to the Eclipse IDE.\n\
```

Comment:
I prefer the current text. Right now it supports App Engine standard environment only. The time to make the message different is when we add support for additional environments, not before.